docs/releases/1.11.txt

```diff
@@ -399,6 +399,10 @@ Miscellaneous
 * The :djadmin:`makemessages` command now requires configured settings, like
   most other commands.
 
+* ``utils.http.is_safe_url()`` accepts a new ``secure`` argument. If
```

Comment:
> I'm not sure this needs to be mentioned in backwards-incompatible changes changes?

Of course not. That wasn't intentional :) Thanks, fixed now!

And usage in django views is moved to separate commits and is also mentioned in release notes.